### DISCUSSION TITLE
Correct way to import the "skipToken"

### DISCUSSION THREAD

**Discussion by User6531** [reduxjs/redux-toolkit#3499] [Community Recipes]
I'm currently using the following dependencies in my project: 
• "vite": 4.1.1
• "@reduxjs/toolkit": 1.9.5
• "react": 18.2.0.

There was an issue I encountered where I received an "undefined" when using the import path `@reduxjs/toolkit/dist/query` in the built version of my project.

```tsx
import { skipToken } from "@reduxjs/toolkit/dist/query";

console.log(skipToken); // undefined ❌
```

However, when I changed the import path to `@reduxjs/toolkit/query/react`, everything worked fine.

```tsx
import { skipToken } from "@reduxjs/toolkit/query/react";

console.log(skipToken); // Symbol(RTKQ/skipToken) ✅
```

Upon reviewing the documentation, I realized that the recommended import path is `@reduxjs/toolkit/query/react`. Unfortunately, during development, I relied on autocomplete and didn't pay attention to the import path, leading to hours spent trying to find the problem.

I hope this information can help someone else save time and avoid similar issues.

**Comment by EskiMojo14** [comment]
yep, you shouldn't ever be importing from /dist (except for internal types, and even then import at your own risk). 

The supported entry points are:

- `@reduxjs/toolkit`
- `@reduxjs/toolkit/react` (new in RTK 2.0, currently in beta)
- `@reduxjs/toolkit/query`
- `@reduxjs/toolkit/query/react`

skipToken is exported from both of the last two, I believe.
